### PR TITLE
Fix performance tests

### PR DESCRIPTION
**Pull request by kitaisreal**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: tests/performance/group_array_sorted.xml [hunk 1]
```diff
@@ -1,27 +1,11 @@
...
-            </values>
-        </substitution>
-    </substitutions>
+    <create_query>CREATE TABLE test (`id` UInt64, `value` UInt64, `text` String ) ENGINE = Memory</create_query>
+    <fill_query>INSERT INTO test SELECT number as id, rand64() as value, toString(number) as text FROM system.numbers_mt LIMIT 10000000</fill_query>
+
+    <query>SELECT groupArraySorted(100000)(id, value) FROM test</query>
+    <query>SELECT groupArraySorted(100000)(text, value) FROM test</query>
+    <query>SELECT groupArraySorted(100000)((id, text), value) FROM test</query>
+    <query>SELECT groupArraySorted(100000)(text) FROM test</query>
```
Comment (Algunenano) [inline tests/performance/group_array_sorted.xml:9]: I think these were meant to test different limit sizes cc @palegre-tiny

Comment (palegre-tiny) [reply tests/performance/group_array_sorted.xml:9]: Yes, that was the idea. Testing performance for difference dataset sizes.

**Comment by alexey-milovidov** [comment]
This PR does not fix the tests. It will be not worse if we merge this PR and continue in another PR.

**Comment by alexey-milovidov** [comment]
Performance tests are broken for 4 months already, I can't stand it.

**Pull request merged**